src/construction.cpp

```diff
@@ -1141,7 +1141,7 @@ void complete_construction( Character *you )
     // This comes after clearing the activity, in case the function interrupts
     // activities
     built.post_special( terp );
-    // npcs will automatically resume backlog, players won't.
```

Comment:
This one looks fine to correct, too

Reply:
I have a soft spot for these kinds of typos specially because they are inside comments, I intentionally left a lot of `wont`s, `thats`s and variants as they are